ERP-BACKEND/.env

```
@@ -0,0 +1,3 @@
+DATABASE_URL=postgresql://erp_user:erp_password@localhost:5432/erp_db
+SECRET_KEY=your-super-secret-key-change-in-production-must-be-32-chars-min
+ACCESS_TOKEN_EXPIRE_MINUTES=1440
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ACCESS_TOKEN_EXPIRE_MINUTES key should go before the DATABASE_URL key

(UnorderedKey)